File: CSharp/Hangover.cs
using Twilio;
using Twilio.Rest.Api.V2010.Account;

//Exit early if any session with my username is found
if (args[0] is null)
{
    return;
}

var twilioAccountSid = Environment.GetEnvironmentVariable("TWILIO_ACCOUNT_SID");
var authToken = Environment.GetEnvironmentVariable("TWILIO_AUTH_TOKEN");

//Phone numbers
const string myNumber = "+xxx";
const string numberOfBoss = "+xxx";

TwilioClient.Init(twilioAccountSid, authToken);

var excuses = await new ChatGpt().GetExcusesToMyBoss();

var rand = new Random().Next(excuses.Length);
var message = $"Gonna work from home. {excuses[rand]}";

//Send a text message
var response = MessageResource.Create(
    body: message,
    from: new Twilio.Types.PhoneNumber(myNumber),
    to: new Twilio.Types.PhoneNumber(numberOfBoss)
);

Console.WriteLine(response.Sid);

File: clojure/hangover.clj
(ns hacker-scripts.hangover
  (:import
    (com.twilio Twilio)
    (com.twilio.rest.api.v2010.account Message)
    (com.twilio.type PhoneNumber)))

(def acc-sid "my twilio account SID")
(def acc-tkn "my twilio secret token")

(def my-num (PhoneNumber. "+10001112222"))
(def boss-num (PhoneNumber. "+19998887777"))

(def reasons ["Receiving delivery"
              "Waiting for repairman"
              "Nasty cold"])

(defn twilio-init []
  (Twilio/init acc-sid acc-tkn))

(defn send-sms [to-num from-num message]
  (.. Message (creator to-num from-num message) create))

(def send-sms-boss (partial send-sms boss-num my-num))

(defn hangover []
  (twilio-init)
  (let [message (rand-nth reasons)]
    (send-sms-boss message)))
